store_app/views.py

```diff
@@ -45,3 +45,22 @@ def get_context_data(self, **kwargs):
         context['products_for_month'] = products_for_month
         context['products_for_year'] = products_for_year
         return context
+
+
+def edit_product(request, product_id):
```

Comment:
Это первый шаг к тому, чтобы сделать универсальную форму, которая бы изменяла и добавляла новый продукт
```suggestion
def edit_product(request, product_id=None):
```